### ISSUE TITLE
coredump_disable_backtraces & coredump_disable_storage rules ignore overrides files

### ISSUE THREAD

**Issue by JGoutin** [ComplianceAsCode/content#10686]
#### Description of problem:

Apply to the following rules:
* coredump_disable_backtraces
* coredump_disable_storage

Theses rules only check the `/etc/systemd/coredump.conf` file, but  should also check overrides files in `/etc/systemd/coredump.conf.d/`.

The overrides files can be used to both disable or enable the required configuration.

#### SCAP Security Guide Version:

Benchmark ID: xccdf_org.ssgproject.content_benchmark_FEDORA
Benchmark version: 0.1.67
Profile ID: xccdf_org.ssgproject.content_profile_ospp

#### Operating System Version:

Fedora 38

#### Steps to Reproduce:

1. Ensure `/etc/systemd/coredump.conf` is unmodified.
2. Create an override file like `/etc/systemd/coredump.conf.d/disabled.conf` with  the following content:
   ```
   [Coredump]
   Storage=none
   ProcessSizeMax=0
   ```
3. This command can be used to check the configuration is applied: `systemd-analyze cat-config /etc/systemd/coredump.conf`
4. Run `oscap xccdf eval --profile xccdf_org.ssgproject.content_profile_ospp /usr/share/xml/scap/ssg/content/ssg-fedora-ds.xml`

#### Actual Results:

The rules check FAIL

#### Expected Results:

The rules check PASS

**Comment by Mab879** [comment]
It appears this is fixed now.

**Comment by jan-cerny** [comment]
Yes, the OVALs for these rules actually contain a check for the .d directory.
